app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -78,20 +78,20 @@ def diagnostic_codes
   end
 
   def prepare_for_evss!
-    return if pending_eps? || disability_not_service_connected?
+    return if pending_eps? || disabilities_not_service_connected?
 
     save_metadata(forward_to_mas_all_claims: true) if Flipper.enabled?(:rrd_mas_all_claims_tracking) &&
                                                       !single_issue_hypertension_cfi?
 
     if Flipper.enabled?(:rrd_mas_disability_tracking) && single_disability_eligible_for_mas?
       save_metadata(forward_to_mas: true)
-      insert_classification_codes
+      insert_classification_codes unless Flipper.enabled?(:rrd_mas_all_claims_notification)
```

Comment:
Since the relationship between the flipper flag name and `insert_classification_codes` isn't obvious, how about adding a comment that says something like "MAS doesn't want contention classification codes to be set". I believe we added this for only MAS.